package.json

```diff
@@ -23,6 +23,7 @@
     "@types/yargs": "^8.0.1",
     "chai": "^4.1.2",
     "mocha": "^4.1.0",
+    "sane": "^2.5.0",
```

Comment:
Is this used anywhere? The tests (for the `fsevents` test?)

Reply:
Yeah, used for the `fsevents` test 👍